.github/workflows/update-tags.yaml

```
@@ -0,0 +1,15 @@
+name: Keep the versions up-to-date
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  actions-tagger:
+    runs-on: windows-latest
```

```suggestion
    runs-on: ubuntu-latest
```
Skimming through the action it just uses the github API, that should work on any os